docs/getting-started/raster/raster_step-by-step.md

```diff
@@ -102,17 +102,25 @@ url:
 
 body (XML):
 <?xml version="1.0" encoding="UTF-8"?>
-<csw:GetRecords xmlns:csw="http://www.opengis.net/cat/csw/2.0.2" service="CSW" maxRecords="1"  startPosition="1"  outputSchema="http://schema.mapcolonies.com/raster" version="2.0.2" xmlns:mc="http://schema.mapcolonies.com/raster" >
+<csw:GetRecords xmlns:csw="http://www.opengis.net/cat/csw/2.0.2" service="CSW" maxRecords="10"  startPosition="1"  outputSchema="http://schema.mapcolonies.com/raster" version="2.0.2" xmlns:mc="http://schema.mapcolonies.com/raster" >
   <csw:Query typeNames="mc:MCRasterRecord">
    <csw:ElementSetName>full</csw:ElementSetName>
     <csw:Constraint version="1.1.0">
       <Filter xmlns="http://www.opengis.net/ogc">
-        <PropertyIsLike wildCard="%" singleChar="_" escapeChar="\\">
-          <PropertyName>mc:productId</PropertyName>
-          <!-- ****** INSERT LAYER LAYER ID START ********************** -->
-          <Literal>bluemarble_5km</Literal>
-          <!-- ****** INSERT LAYER LAYER ID END ************************ -->
-        </PropertyIsLike>
+  		<And>
+	        <PropertyIsEqualTo matchCase="true">
+	          <PropertyName>mc:productId</PropertyName>
+            <!-- ****** INSERT LAYER LAYER ID START ********************** -->
+	          <Literal>bluemarble_5km</Literal>
```

Comment:
put the values matching for best orthophoto and not bluemarble

Reply:
done